aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -175,7 +175,11 @@ Tensor& set_storage_quantized_(
   auto* self_ = self.unsafeGetTensorImpl();
   self_->set_storage_keep_dtype(storage);
   self_->set_storage_offset(storage_offset);
-  self_->set_sizes_and_strides(sizes, strides);
+  if(strides.data() == nullptr){
```

Comment:
when does this happen? should we make sure the strides are always not nullptr?

Reply:
https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/serialization/unpickler.cpp#L516
When the unpickler try to deserialize a `qtensor`, it use `.set(storage, 0, {}, {})`, where strides is not provided here.